src/main/java/ch/ivyteam/ivy/maven/MavenDependencyCleanupMojo.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2018 AXON Ivy AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ch.ivyteam.ivy.maven;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+import ch.ivyteam.ivy.maven.engine.MavenProjectBuilderProxy;
+
+/**
+ * Copy maven dependencies to a specific folder.
+ * 
+ * @since 9.2.0
+ */
+@Mojo(name=MavenDependencyCleanupMojo.GOAL, requiresDependencyResolution=ResolutionScope.COMPILE)
+public class MavenDependencyCleanupMojo extends AbstractProjectCompileMojo
+{
+  public static final String GOAL = "maven-dependency-cleanup";
+  
+  /** 
+   * Set to <code>true</code> to bypass the copy of <b>maven dependencies</b>.
+   */
+  @Parameter(property="ivy.mvn.dep.cleanup.skip", defaultValue="false")
+  boolean skipMvnDependencyCleanup;
+  
+  @Override
+  protected void compile(MavenProjectBuilderProxy projectBuilder) throws Exception
+  {
+    if (skipMvnDependencyCleanup)
+    {
+      return;
+    }
+    var mvnLibDir = project.getBasedir().toPath().resolve("lib").resolve("mvn-deps");
+    getLog().info("Deleting " + mvnLibDir.toString());

```

Comment:
no info logs for expected behaviour please :)

Reply:
If I remove this info log (or change it to debug), there will be no log at all for this mojo. This will product a similar log as the normal clean plugin produces (see picture below). So I like this, but I don't have a strict meaning about this and can change it to debug.

Reply:
ok, yes I see that your output matches perfectly the clean plugin impl. .. So I'd vote to keep it as is :+1: